crystal/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*.cr]
```

Comment:
would having an .editorconfig file at the repository level be wanted?

Reply:
`crystal init` added it; I figured I should keep it but I wouldn't object to removing it